README.md

```diff
@@ -598,11 +613,14 @@ API | Description | Auth | HTTPS | Link |
 | Bay Area Rapid Transit | Stations and predicted arrivals for BART | `apiKey` | No | [Go!](http://api.bart.gov) |
 | Community Transit | Transitland API | No | Yes | [Go!](https://github.com/transitland/transitland-datastore/blob/master/README.md#api-endpoints) |
 | Goibibo | API for travel search | `apiKey` | Yes | [Go!](https://developer.goibibo.com/docs) |
+| GraphHopper | A-to-B routing with turn-by-turn instructions | `apiKey` | Yes | [Go!](https://graphhopper.com/api/1/docs/) |
 | Icelandic APIs | Open APIs that deliver services in or regarding Iceland | No | Yes | [Go!](http://docs.apis.is/) |
 | Indian Railways | Indian Railways Information | `apiKey` | No | [Go!](http://api.erail.in/) |
 | Izi | Audio guide for travellers | `apiKey` | Yes | [Go!](http://api-docs.izi.travel/) |
+| Lyft | Request and lookup ride data | `apiKey` | Yes | [Go!](https://www.lyft.com/developers) |
```

Comment:
This is the one of the larger gray area of this project, where an API itself is free, but the service behind it is not. For now, we are not including services like this in the project.

Reply:
👍 Understood. We should remove Uber then as well, right?

Reply:
I went ahead and removed Uber too.